### PR TITLE
Pause and resume campaigns in Adzerk

### PR DESCRIPTION
**Pull request by zeantsoi**
:eyeglasses: @dwick @bsimpson63 

In conjunction with [reddit-public #4341](https://github.com/reddit/reddit-public/pull/4341).

### CODE REVIEW

**Comment by dwick** [comment]
:fish

**Comment by dwick** [comment]
:fish:

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -254,8 +254,9 @@ def update_flight(link, campaign, az_campaign):
         'CampaignId': az_campaign.Id,
         'PriorityId': g.az_selfserve_priorities[campaign.priority_name],
         'IsDeleted': False,
-        'IsActive': (promote.charged_or_not_needed(campaign) and
-                     not (campaign._deleted or campaign_overdelivered)),
+        'IsActive': (not campaign.paused or
```
Comment (bsimpson63) [inline reddit_adzerk/adzerkpromote.py]: Won't this be `True` as long as the campaign is not paused, regardless of the following checks?

**Comment by bsimpson63** [comment]
:nail_care:

**Comment by zeantsoi** [comment]
:haircut:

File: reddit_adzerk/adzerkpromote.py [hunk 2]
```diff
@@ -566,6 +567,10 @@ def edit_campaign(link, campaign):
 def delete_campaign(link, campaign):
     update_adzerk(link, campaign)
 
+@hooks.on('promote.toggle_pause_campaign')
```
Comment (bsimpson63) [inline reddit_adzerk/adzerkpromote.py]: up to you, but the reddit-public code could just use the `promote.edit_campaign` hook

Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Makes sense to me; I've removed this hook and pointed to `edit_campaign`.

**Comment by bsimpson63** [comment]
:fish:

**Comment by dwick** [comment]
re :fish:

**Comment by zeantsoi** [comment]
8e048acf4372565bf3d622f5ff5783965fb6156a

**Pull request closed without merging**